Dockerfile

```diff
@@ -26,6 +26,9 @@ RUN apt-get update && \
 
 COPY CHANGES.rst README.rst setup.py /code/
 COPY reana_workflow_controller/version.py /code/reana_workflow_controller/
+ARG nocache
+RUN echo $nocache
```

Comment:
Could $nocache be removed?

Reply:
absolutly! I was using it locally to not wait more on the rebuilds!